c2corg_api/tests/views/test_user.py

```diff
@@ -285,3 +285,40 @@ def test_restricted_request_unauthenticated(self):
 
     def _add_test_data(self):
         pass
+
+    def test_read_and_update_account_info(self):
```

Comment:
I would appreciate some more tests (e.g. failures) because changing the email and password is critical.